public/src/cards.js

```diff
@@ -95,6 +95,8 @@ let events = {
       pack[name] = 1
     }
     App.update()
+    if (document.hidden && App.state.beep)
```

Comment:
Regarding the focusing on another window, could [this](https://developer.mozilla.org/en-US/docs/Web/API/document.hasFocus) work:

`if (App.state.beep && (document.hidden || !document.hasFocus))`